### PR TITLE
Translate strict type checking options into zh

### PR DESCRIPTION
**Pull request by Kingwl**
https://github.com/microsoft/TypeScript-Website/issues/296

### CODE REVIEW

**Comment by garfeildma** [comment]
LGTM

File: packages/tsconfig-reference/copy/zh/options/alwaysStrict.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+display: "始终严格模式"
+oneline: "保证始终生成 “use strict”"
+---
+
+保证你的文件在 ECMAScript 严格模式下被解析，并且在每个编译后的文件中生成 “use strict”
```
Comment (aladdin-add) [inline packages/tsconfig-reference/copy/zh/options/alwaysStrict.md:6]: 这里的引号是全角的么？？

其它文件也有很多全角的引号

File: packages/tsconfig-reference/copy/zh/options/strictBindCallApply.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+当开启后，TypeScript 将检查内置函数 `bind`，`call` 和 `apply` 是否接受了被调用函数的正确参数：
+
+
+```ts twoslash
+// @strictBindCallApply: true
+// @errors: 2345
+
+// With strictBindCallApply on
```
Comment (aladdin-add) [inline packages/tsconfig-reference/copy/zh/options/strictBindCallApply.md]:
```suggestion
// 启用 strictBindCallApply 选项
```

**Review by aladdin-add — CHANGES_REQUESTED** [review]

**Comment by Kingwl** [comment]
@laosb PTAL. thanks!

**Comment by orta** [comment]
:wave: hey folks, I've handled the merge conflict on this PR - if this PR is good to go, lets try get it in before friday.

Otherwise I'll merge this PR at the end of the week, as it looks like a LGTM to me,  because we're moving these PRs into their own repos re: https://github.com/microsoft/TypeScript-Website/issues/1480

**Comment by orta** [comment]
Well, that message could be improved.

**Comment by Kingwl** [comment]
Thanks!

I think It's basically done. so:

LGTM

**Pull request merged**